### PR TITLE
Remove zoomIn flag from ViewWorld::ZoomROIs::ChangeZoom

### PR DESCRIPTION
**Pull request by qarkai**
Split `ChangeZoom()` to `zoomIn()` and `zoomOut()`.

### CODE REVIEW

File: src/fheroes2/kingdom/view_world.cpp [hunk 1]
```diff
@@ -420,14 +420,23 @@ bool ViewWorld::ZoomROIs::ChangeCenter( const fheroes2::Point & centerInPixels )
...
 }
 
-bool ViewWorld::ZoomROIs::ChangeZoom( const bool zoomIn, const bool cycle )
+bool ViewWorld::ZoomROIs::changeZoom( const ZoomLevel newLevel )
 {
-    ViewWorld::ZoomLevel newLevel = zoomIn ? GetNextZoomLevel( _zoomLevel, cycle ) : GetPreviousZoomLevel( _zoomLevel, cycle );
-    if ( newLevel == _zoomLevel ) {
-        return false;
-    }
+    bool changed = ( newLevel != _zoomLevel );
```
Comment (ihhub) [inline src/fheroes2/kingdom/view_world.cpp:425]: We could make this variable `const`.

File: src/fheroes2/kingdom/view_world.h [hunk 2]
```diff
@@ -58,7 +58,8 @@ class ViewWorld
     {
         ZoomROIs( const ZoomLevel zoomLevel, const fheroes2::Point & centerInPixels );
 
-        bool ChangeZoom( const bool zoomIn, const bool cycle = false );
+        bool zoomIn( const bool cycle = false );
+        bool zoomOut( const bool cycle = false );
```
Comment (ihhub) [inline src/fheroes2/kingdom/view_world.h:62]: Can we please avoid using default values for input arguments?

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @qarkai , I left just 2 comments here. Could you please take a look?

File: src/fheroes2/kingdom/view_world.h (hunk 2 above)
Comment (qarkai) [reply src/fheroes2/kingdom/view_world.h]: As you wish.

**Comment by ihhub** [comment]
@qarkai , thank you very much for this code cleanup!

**Pull request merged**